### PR TITLE
[libatomic-ops] Disable docs

### PR DESCRIPTION
**Pull request by Thomas1664**
**Describe the pull request**

- #### What does your PR fix?
  Disable installing docs
  License, homepage

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  unchanged

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

### CODE REVIEW

**Comment by Thomas1664** [comment]
@FrankXie05 Why requires:author-response?

**Comment by FrankXie05** [comment]
@Thomas1664 The port has two license.

**Review by FrankXie05 — APPROVED** [review]

File: ports/libatomic-ops/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,10 @@
 {
   "name": "libatomic-ops",
   "version": "7.7.0",
+  "port-version": 1,
   "description": "The atomic_ops project (Atomic memory update operations portable implementation)",
+  "homepage": "https://github.com/ivmai/libatomic_ops",
+  "license": "GPL-2.0",
```
[the remark this answers is not in the document]
Comment (dg0yt) [reply ports/libatomic-ops/vcpkg.json]: It is not really `OR`:
~~~
A few files are covered by the GNU General Public License.  (See file
"COPYING".)  This applies only to the test code and the atomic_ops_gpl
portion of the library.  Thus, atomic_ops_gpl should generally not be
linked into proprietary code.  (This distinction was motivated by patent
considerations.)
~~~
So if `atomic_ops_gpl`, there is only GPL left. Unfortunately, I don't see an easy way to track this in `license` fields with anything but `null` (i.e. "read the terms").

[the remark this answers is not in the document]
Comment (FrankXie05) [reply ports/libatomic-ops/vcpkg.json]: If  the part of source code involves a license, I  think it is not should be added to the port. 
sorry this is my problem. @Thomas1664 
Thanks dg0yt

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**